lib/flowViewport/sceneIndex/fvpRenderIndexProxyFwd.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef FVP_RENDER_INDEX_PROXY_FWD_H
+#define FVP_RENDER_INDEX_PROXY_FWD_H
+
+#include "flowViewport/api.h"
+
+
+namespace FVP_NS_DEF {
+
+class RenderIndexProxy;
```

Comment:
What is the added value of this header?  You're not defining any additional types.  A client of RenderIndexProxy can do this themselves.  Unless you're planning to add type definitions for RenderIndexProxy related types (e.g. a vector of them, or a pointer to them), I would remove this file.